bin/init.js

```diff
@@ -30,56 +33,284 @@ const info = {
+
 			rl.question( 'Confirm the Theme Details (y/n) ', ( confirm ) => {
+
 				if ( 'n' === confirm.toLowerCase() ) {
 					console.log( info.warning( '\nTheme Setup Cancelled.\n' ) );
-					process.exit( 0 );
+					rl.close();
 				}
 
 				initTheme( themeInfo );
 
-				rl.question( 'Would you like to run the theme cleanup? (y/n) ', ( cleanup ) => {
-					if ( 'n' === cleanup.toLowerCase() ) {
-						console.log( info.warning( '\nExiting without running theme cleanup.\n' ) );
-						process.exit( 0 );
+				// Remove theme initialization scripts.
+				modifyComposerJson();
+				modifyPackageJson();
+
+				rl.question( 'Would you like to initialize git (Note: It will delete any `.git` folder already in current directory)? (y/n) ', async ( initialize ) => {
+					if ( 'n' === initialize.toLowerCase() ) {
+						console.log( info.warning( '\nExiting without initializing GitHub.\n' ) );
+						await askQuestionForHuskyInstallation();
+					} else {
+						await initializeGit()
 					}
-					runThemeCleanup();
-					rl.close();
+					themeCleanupQuestion();
 				} );
 			} );
 		} );
 	} );
 } else if ( ( args.includes( '--clean' ) || args.includes( '-c' ) ) && 1 === args.length ) {
+	themeCleanupQuestion();
+} else {
+	console.log( info.error( '\nInvalid arguments.\n' ) );
+	rl.close();
+}
+rl.on( 'close', () => {
+	process.exit( 0 );
+} );
+
+/**
+ * Modify composer.json file.
+ *
+ * @return {void}
+ */
+const modifyComposerJson = () => {
+	console.log( info.message( '\nRemoving post-install-cmd script from the composer.json...' ) );
+	const composerJsonPath = path.resolve( getRoot(), 'composer.json' );
+
+	if ( ! fs.existsSync( composerJsonPath ) ) {
+		return;
+	}
+
+	const composerJson = require( composerJsonPath );
+
+	try {
+		// Remove scripts.
+		delete composerJson.scripts['post-install-cmd'];
+
+		// Commit the changes to file.
+		fs.writeFileSync( composerJsonPath, JSON.stringify( composerJson, null, 2 ) );
+		console.log( info.success( '\ncomposer.json updated successfully!' ), '✨' );
+	} catch ( error ) {
+		console.log( info.error( `Error while updating composer.json: ${ error.message }` ) );
+	}
+}
+
+/**
+ * Modify package.json file.
+ *
+ * @return {void}
+ */
+const modifyPackageJson = () => {
+	console.log( info.message( '\nRemoving init script from the package.json...' ) );
+	const packageJsonPath = path.resolve( getRoot(), 'package.json' );
+
+	if ( ! fs.existsSync( packageJsonPath ) ) {
+		return;
+	}
+
+	const packageJson = require( packageJsonPath );
+
+	try {
+		delete packageJson.scripts['init'];
+
+		if ( ! packageJson.scripts['prepare'] ) {
+			return;
+		}
+
+		const prepareScript = packageJson.scripts['prepare'];
+
+		// Check if 'npm run init' is part of the prepare script.
+		if ( ! prepareScript.includes('npm run init') ) {
+			return;
+		}
+
+		// Split the prepare script into an array of individual scripts.
+		const prepareScriptArray = prepareScript.split( '&&' ).map( ( script ) => script.trim() );
```

Comment:
Not exactly. Right now we only have `npm run init` but in future if we decide to add some other command in prepare. Then we might want to persist it so this will make sure we only remove `npm run init`.